### PR TITLE
fix(container): fix cookie modal when ghostery on

### PR DESCRIPTION
**Pull request by qpavy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-112556
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~~[ ] Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

## Description

Because ghostery plugin is hiding automatically some content depending of his id or class, we have to change the modal class in order to see the description

### CODE REVIEW

**Review by antonymarion — DISMISSED** [review]

**Review by darsene — APPROVED** [review]

File: packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ const CookiePolicy = ({ shell }: Props): JSX.Element => {
       animation={false}
     >
       <Modal.Body>
-        <div className="cookie-policy-modal p-1">
+        <div className="cookie-policy-modal-description p-1">
```
Comment (anooparveti) [inline packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:81]: **nitpick:** You also need to change the class name in css/scss file no?

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx (hunk 1 above)
Comment (qpavy) [reply packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:81]: I didn't find this class in css / scss

Comment (lionel95200x) [reply packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:81]: `packages/manager/modules/cookie-policy/src/style.scss` Line 8 ?

Comment (qpavy) [reply packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:81]: I don't think this is used by container. Container cookie modal style come from `packages/manager/apps/container/src/cookie-policy/cookie-modal.module.scss` I don't think the react stack is using old angularjs module scss file

Comment (qpavy) [reply packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:81]: or maybe it does but if it does, is it normal ?

Comment (qpavy) [reply packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:81]: but to be be more precise, the react component is using the packages/manager/modules/cookie-policy/src/style.scss style file which is the same as packages/manager/modules/cookie-policy/src/style.scss without the class selector

**Review by darsene — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by lionel95200x — APPROVED** [review]

**Pull request merged**